netdata-installer.sh

```diff
@@ -621,6 +624,72 @@ bundle_libwebsockets() {
 
 bundle_libwebsockets
 
+# -----------------------------------------------------------------------------
+
+build_jsonc() {
+  pushd "${1}" > /dev/null || exit 1
+  run env CFLAGS= CXXFLAGS= LDFLAGS= cmake .
+  run env CFLAGS= CXXFLAGS= LDFLAGS= make
+  popd > /dev/null || exit 1
+}
+
+copy_jsonc() {
+  target_dir="${PWD}/externaldeps/jsonc"
+
+  run mkdir -p "${target_dir}" "${target_dir}/json-c" || return 1
+
+  run cp "${1}/json-c.a" "${target_dir}/json-c.a" || return 1
+  run cp -r "${1}/*.h" "${target_dir}/json-c" || return 1
+}
+
+bundle_jsonc() {
```

Comment:
Where is the pkg-config check to see if it is already installed?

Reply:
Next line down, it's the third item in the conditional that exits early if it all passes.

Reply:
`&& pkg-config json-c ; then`
```
  if [ -z "${NETDATA_DISABLE_CLOUD}" ] && [ -z "${NETDATA_BUILD_JSON_C}" ] && pkg-config json-c ; then
    return 0
  fi
```

Reply:
And it seems to be working as I had to comment those 3 lines out to emulate system without json-c

Reply:
Github is too narrow for me to see it :)